tests/sentry/utils/test_concurrent.py

```diff
@@ -108,10 +106,6 @@ def test_timed_future_success():
         future.set_result(None)
         assert future.get_timing() == (1.0, 2.0)
 
-    with timestamp(3.0):
-        future.set_result(None)
-        assert future.get_timing() == (1.0, 3.0)
-
```

Comment:
Should we change this to make sure that the timing doesn't change if we call `future.set_result` again here?